shoppingcard.css

```diff
@@ -0,0 +1,142 @@
+* {
+
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+
+.container {
+    margin-top: 30px;
+    max-width: 1000px;
+}
```

Comment:
instead of px, use rem. for width use vw.
why do you have to give 1000px? Make it `width: 100%`